bootcamps/uk/bootcamps.md

```diff
@@ -59,4 +59,7 @@
     * Ongoing career support
     * See more details about each course [here](https://www.manchestercodes.com/study-and-courses/courses)
     
-
+## Scotland
```

Comment:
Is there a free or apprenticeship version of this bootcamp? The idea is to have just free bootcamps on here, rather than paid ones. @crgrieve

Reply:
I will make this change just to merge this PR